extensions/azurecore/src/azurecore.d.ts

```diff
@@ -76,10 +80,16 @@ declare module 'azurecore' {
 		provideResources(): azureResource.IAzureResourceProvider[];
 
 		runGraphQuery<T extends azureResource.AzureGraphResource>(account: azdata.Account, subscriptions: azureResource.AzureResourceSubscription[], ignoreErrors: boolean, query: string): Promise<ResourceQueryResult<T>>;
+		makeHttpGetRequest(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, ignoreErrors: boolean, url: string): Promise<HttpGetRequestResult>;
 	}
 
 	export type GetSubscriptionsResult = { subscriptions: azureResource.AzureResourceSubscription[], errors: Error[] };
 	export type GetResourceGroupsResult = { resourceGroups: azureResource.AzureResourceResourceGroup[], errors: Error[] };
+	export type GetSqlManagedInstancesResult = { resources: azureResource.AzureGraphResource[], errors: Error[] };
+	export type GetSqlServersResult = {resources: azureResource.AzureGraphResource[], errors: Error[]};
+	export type GetSqlVMServerResult = {resources: azureResource.AzureGraphResource[], errors: Error[]};
+	export type GetStorageAccountResult = {resources: azureResource.AzureGraphResource[], errors: Error[]};
 
 	export type ResourceQueryResult<T extends azureResource.AzureGraphResource> = { resources: T[], errors: Error[] };
+	export type HttpGetRequestResult = { response: any, errors: Error[] };
```

Comment:
It would be great to have an actual type for this instead of any

Reply:
This is a generic HTTP request and can have many different types of responses.